neurovault/apps/statmaps/management/commands/benchmark.py

```diff
@@ -0,0 +1,104 @@
+from django.core.management.base import BaseCommand, CommandError
+from neurovault.apps.statmaps.tests.utils import clearDB
+from neurovault.apps.statmaps.models import Comparison, Similarity, User, Collection, Image
+from neurovault.apps.statmaps.utils import get_existing_comparisons
+from neurovault.apps.statmaps.tests.utils import  save_statmap_form
+
+import os
+import gc
+import timeit
+import numpy as np
+
+class Timer:
+    def __init__(self, timer=None, disable_gc=False, verbose=True):
+        if timer is None:
+            timer = timeit.default_timer
+        self.timer = timer
+        self.disable_gc = disable_gc
+        self.verbose = verbose
+        self.start = self.end = self.interval = None
+    def __enter__(self):
+        if self.disable_gc:
+            self.gc_state = gc.isenabled()
+            gc.disable()
+        self.start = self.timer()
+        return self
+    def __exit__(self, *args):
+        self.end = self.timer()
+        if self.disable_gc and self.gc_state:
+            gc.enable()
+        self.interval = self.end - self.start
+        if self.verbose:
+            print('time taken: %f seconds' % self.interval)
+
+class Command(BaseCommand):
+    args = '<times_to_run>'
+    help = 'bench'
+
+    def handle(self, *args, **options):
+        # TODO: check if /bench folder exists. If not, donwload the full dataset.
```

Comment:
No problem with this. The candidate images are: MNI templated, unthresholded and modality=fMRI-BOLD. Something else?


I've a non-benchmark-related "problem" with this;
Candidate images with previous constraints are like 5000. This is not a problem, but 1 collection of the candidates has like 4000 candidate images itself:

In one hand, http://neurovault.org/collections/503/ has 4,792 images, from a total of 11600 images in Neurovault, that makes almost half of the whole db. Downloading for testing purposes it's ok, but shall be this Collection taken into account when testing ANN queries? is biasing all the results. 

thanks chris